src/main/java/ch/heigvd/pro/b04/polls/ServerPollIdentifier.java

```diff
@@ -24,6 +25,7 @@
   private long idPoll;
 
   @ManyToOne
+  @Setter
```

Comment:
This is not needed - you could use the `Builder` to do exactly the same thing.

Reply:
I'll modify it

Reply:
You mean to rebuild it entierly ?

Reply:
This class has a Builder constructor. Why not use it ?